### PR TITLE
Submit Quicklinks example, and update notify-click-links-i18n to use placeholders

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

File: quicknote/popup/quicknote.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+html,body {
+  margin: 0;
+}
+
+html {
+  font-family: sans-serif;
+  font-size: 10px;
+  background: background: rgb(240,240,240);
```
Comment (wbamberg) [inline quicknote/popup/quicknote.css]: typo

File: quicknote/popup/quicknote.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  height: 300px;
+}
+
+body {
+  width: 300px;
+  background: rgb(240,240,240);
+  margin: 0 auto;
```
Comment (wbamberg) [inline quicknote/popup/quicknote.css]: I'm not sure why you are defining margin at line 8 and again here. Why not move the declaration at 8 into the rule for html? Is there a thing I'm missing here?

File: quicknote/popup/quicknote.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+addBtn.addEventListener('click', addNote);
+clearBtn.addEventListener('click', clearAll);
+
+/* display previously-saved stored notes on startup */
+
+initialize();
+
+function initialize() {
+  chrome.storage.local.get(null,function(results) {
+    var noteKeys = Object.keys(results);
```
Comment (wbamberg) [inline quicknote/popup/quicknote.js]: should this check for runtime.lastError?

File: quicknote/popup/quicknote.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      displayNote(noteTitle,noteBody);
+      storeNote(noteTitle,noteBody);
+    }
+  })
+}
+
+/* function to store a new note in storage */
+
+function storeNote(title, body) {
+  chrome.storage.local.set({ [title] : body }, function() {
```
Comment (wbamberg) [inline quicknote/popup/quicknote.js]: and also here?

File: quicknote/manifest.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  "description": "Allows the user to make quick notes by clicking a button and entering text into the resulting popup. The notes are saved in storage. See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Examples#quicknote",
+  "icons": {
+    "48": "icons/quicknote-48.png"
+  },
+
+  "applications": {
+    "gecko": {
+      "id": "quicknote@mozilla.org",
+      "strict_min_version": "45.0.0"
+    }
```
Comment (wbamberg) [inline quicknote/manifest.json]: should be "45.0" (sorry, this is my fault)

**Comment by wbamberg** [comment]
Thanks for submitting this Chris, it looks great. I just had a few little comments....

File: quicknote/popup/quicknote.js (hunk 4 above)
Comment (chrisdavidmills) [inline quicknote/popup/quicknote.js]: Check added. Let me know if this is ok, and if I need to do anything else.

File: quicknote/popup/quicknote.css (hunk 2 above)
Comment (chrisdavidmills) [inline quicknote/popup/quicknote.css]: Slackness fixed!

**Comment by wbamberg** [comment]
Thanks @chrisdavidmills !

**Pull request merged**